src/source/examples/post-campaign-example.rst

```diff
@@ -0,0 +1,29 @@
+.. code-block:: javascript
+
+    POST /api/sellside/campaign
+    Accept: application/sellside.campaign-v2+json
+    Content-Type: application/sellside.campaign-v2+json
+    {}
```

Comment:
this is currently the only way to create a campaign, an empty one? No way to create with status, budgets and vendorID in the POST call?

Reply:
nope `A minimum payload is an empty JSON, as none of the fields are mandatory.`  `The full campaign request payload model that can be provided, has the following structure:` so you can indeed provide the full payload if you wish so.